os/arch/arm/src/common/up_internal.h

```diff
@@ -255,8 +255,8 @@ EXTERN uint32_t g_dabtstackbase;	/* Initial top of data abort stack */
  *    of _data.  like:  uint32_t *pdata = &_sdata;
  */
 
-EXTERN uint32_t _stext;			/* Start of .text */
-EXTERN uint32_t _etext;			/* End_1 of .text + .rodata */
+EXTERN uint32_t _stext;			/* Start of .text in Flash region */
+EXTERN uint32_t _etext;			/* End_1 of .text + .rodata in Flash region */
```

Comment:
We need to add the new linker variables in this file. 
This file will be included in many other places to provide extern definition for the linker variables.